test/gtest/ucs/test_event_set.cc

```diff
@@ -163,11 +174,14 @@ UCS_TEST_F(test_event_set, ucs_event_set_tmo_thread) {
     EXPECT_EQ(UCS_OK, status);
     EXPECT_TRUE(event_set != NULL);
 
-    status = ucs_event_set_add(event_set, pipefd[0], UCS_EVENT_SET_EVREAD);
+    status = ucs_event_set_add(event_set, pipefd[0], UCS_EVENT_SET_EVREAD,
+                               NULL);
     EXPECT_EQ(UCS_OK,status);
 
-    status = ucs_event_set_wait(event_set, 20, event_set_func3, NULL);
-    EXPECT_EQ(UCS_OK, status);
+    nread = ucs_event_set_wait(event_set, MAX_EVENT_SET_SIZE, 20,
+                               event_set_func3, NULL, &status);
+    EXPECT_EQ(0, nread);
+//    EXPECT_EQ(NULL, status);
```

Comment:
pls, remove

Reply:
I checked UCS_STATUS instead of NULL check